### PR TITLE
Add security notes to the URL imports documentation.

### PR DESCRIPTION
**Pull request by leerob**

### CODE REVIEW

File: docs/api-reference/next.config.js/url-imports.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ import { a, b, c } from 'https://example.com/modules/some/module.js'
 
 URL Imports can be used everywhere normal package imports can be used.
 
+## Security Model
+
+URL imports provide an opportunity for stronger security of sharing ES modules than what exists today.
```
Comment (styfle) [inline docs/api-reference/next.config.js/url-imports.md:32]: I’m not really sure what this means: “exists today”. Can we remove this sentence?

File: docs/api-reference/next.config.js/url-imports.md [hunk 2]
```diff
@@ -27,6 +27,12 @@ import { a, b, c } from 'https://example.com/modules/some/module.js'
...
+
+This feature is being designed with **security as the top priority**. To start, we added an experimental flag forcing you to explicitly allow the domains you accept URL imports from. Next, we're working to take this further by limiting URL imports to execute in the browser sandbox using the [Edge Runtime](/docs/api-reference/edge-runtime.md). This runtime is used by [Middleware](/docs/middleware.md) as well as [Next.js Live](https://vercel.com/live).
```
Comment (styfle) [inline docs/api-reference/next.config.js/url-imports.md:34]: Can we avoid starting a sentence with “Next”? How about “In the future” instead?

File: docs/api-reference/next.config.js/url-imports.md (hunk 1 above)
Comment (leerob) [reply docs/api-reference/next.config.js/url-imports.md:32]: Exists today with npm, we can drop it though 👍

File: docs/api-reference/next.config.js/url-imports.md (hunk 2 above)
Comment (leerob) [inline docs/api-reference/next.config.js/url-imports.md:34]:
```suggestion
This feature is being designed with **security as the top priority**. To start, we added an experimental flag forcing you to explicitly allow the domains you accept URL imports from. We're working to take this further by limiting URL imports to execute in the browser sandbox using the [Edge Runtime](/docs/api-reference/edge-runtime.md). This runtime is used by [Middleware](/docs/middleware.md) as well as [Next.js Live](https://vercel.com/live).
```

**Review by timneutkens — APPROVED** [review]

File: docs/api-reference/next.config.js/url-imports.md [hunk 3]
```diff
@@ -27,6 +27,10 @@ import { a, b, c } from 'https://example.com/modules/some/module.js'
 
 URL Imports can be used everywhere normal package imports can be used.
 
+## Security Model
+
+This feature is being designed with **security as the top priority**. To start, we added an experimental flag forcing you to explicitly allow the domains you accept URL imports from. We're working to take this further by limiting URL imports to execute in the browser sandbox using the [Edge Runtime](/docs/api-reference/edge-runtime.md). This runtime is used by [Middleware](/docs/middleware.md) as well as [Next.js Live](https://vercel.com/live).
```
Comment (timneutkens) [inline docs/api-reference/next.config.js/url-imports.md:32]: This looks fine, I think it should also explain how files are downloaded and locked during development and not downloaded at runtime.

**Pull request merged**